File: PlaceYourBets/Fixture.vb
Imports System.Net

Public Class Fixture
    Public Property ID() As String
        Get
            Return m_ID
        End Get
        Set(value As String)
            m_ID = Value
        End Set
    End Property
    Private m_ID As String
    Public Property Home_Team() As String
        Get
            Return m_Home_Team
        End Get
        Set(value As String)
            m_Home_Team = Value
        End Set
    End Property
    Private m_Home_Team As String
    Public Property Away_Team() As String
        Get
            Return m_Away_Team
        End Get
        Set(value As String)
            m_Away_Team = Value
        End Set
    End Property
    Private m_Away_Team As String
    Public Property League() As String
        Get
            Return m_League
        End Get
        Set(value As String)
            m_League = Value
        End Set
    End Property
    Private m_League As String
    Public Property [Date]() As String
        Get
            Return m_Date
        End Get
        Set(value As String)
            m_Date = Value
        End Set
    End Property
    Private m_Date As String
    Public Property Kick_Off() As String
        Get
            Return m_Kick_Off
        End Get
        Set(value As String)
            m_Kick_Off = Value
        End Set
    End Property
    Private m_Kick_Off As String
    Public Property Prediction As Integer
        Get
            Return m_prediction
        End Get
        Set(value As Integer)
            m_prediction = value
        End Set
    End Property
    Private m_prediction As String
End Class

File: PlaceYourBets.ConvertedToC#/Fixture.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Xml.Linq;
using System.Net;
namespace PlaceYourBets
{

	public class Fixture
	{
		public string ID {
			get { return m_ID; }
			set { m_ID = value; }
		}
		private string m_ID;
		public string Home_Team {
			get { return m_Home_Team; }
			set { m_Home_Team = value; }
		}
		private string m_Home_Team;
		public string Away_Team {
			get { return m_Away_Team; }
			set { m_Away_Team = value; }
		}
		private string m_Away_Team;
		public string League {
			get { return m_League; }
			set { m_League = value; }
		}
		private string m_League;
		public string Date {
			get { return m_Date; }
			set { m_Date = value; }
		}
		private string m_Date;
		public string Kick_Off {
			get { return m_Kick_Off; }
			set { m_Kick_Off = value; }
		}
		private string m_Kick_Off;
		public int Prediction {
			get { return m_prediction; }
			set { m_prediction = value; }
		}
		private string m_prediction;
	}
}
